src/Mvc/Mvc.RazorPages/src/DependencyInjection/MvcRazorPagesMvcCoreBuilderExtensions.cs

```diff
@@ -14,6 +14,8 @@
 using Microsoft.Extensions.Options;
 using Resources = Microsoft.AspNetCore.Mvc.RazorPages.Resources;
 
+using System.Diagnostics.CodeAnalysis;
```

Comment:
total nit, but seems like style in this codebase is to have using System.* before using Microsoft.*

Reply:
👍 also usings are not separated by newline(s). Right-click in VS and select "Remove and Sort Usings"

Reply:
What is the emacs option? ;)

Fixed.